### ISSUE TITLE
Lualine 

### ISSUE THREAD

**Issue by doctorfree** [neanias/everforest-nvim#2]
I am running into an issue attempting to use the `everforest` theme in `lualine`. If I just set `theme = "everforest" in the lualine setup options as suggested by the README, lualine complains that it doesn't know about the everforest theme and falls back to "auto".

I then copied `lua/lualine/themes/everforest.lua` from this repository to my neovim cconfig as `lua/themes/lualine/everforest.lua` and in my lualine config did:

```lua
theme = require("themes.lualine.everforest")
```
This produced the following error:

```
  Error  04:58:08 PM notify.error lazy.nvim Failed to run `config` for lualine.nvim

...e/.config/nvim-lazyman/lua/themes/lualine/everforest.lua:12: attempt to index local 'palette' (a nil value)

# stacktrace:
  - everforest.lua:12
  - ~/.config/nvim-lazyman/lua/config/lualine.lua:20
  - ~/.config/nvim-lazyman/lua/plugins/ui.lua:135 _in_ **config**
  - /tabline.nvim/lua/tabline.lua:841 _in_ **setup**
  - ~/.config/nvim-lazyman/lua/config/tabline.lua:1
  - ~/.config/nvim-lazyman/lua/plugins/ui.lua:143 _in_ **config**
```

My setup is somewhat convoluted as I am supporting multiple themes and multiple neovim configurations so it is entirely possible the issue is on my end. But, I like everforest and I am glad there is a lua implementation so I wanted to inform you of this potential issue with the lualine support.

The good news is that lualine's "auto" support for everforest is very good. So for now I am just using `theme = "auto"` in my lualine config for everforest.

**Comment by neanias** [comment]
Weird, I'll look into this

**Comment by neanias** [comment]
Fixed in #8
